requirements.txt

```diff
@@ -0,0 +1,114 @@
+aiohttp==3.8.4
```

Comment:
we maintain requirements in pyproject.toml file

Reply:
Oh !, My bad  I used to do every time requirements.txt , Haha